pyanaconda/ui/gui/spokes/advanced_storage.py

```diff
@@ -498,8 +498,13 @@ def setup(self, store, selectedNames, disks):
         for disk in disks:
             paths = [d.name for d in disk.parents]
             selected = disk.name in selectedNames
+            mutable = not disk.protected
 
-            store.append([True, selected, not disk.protected,
+            if disk.mode != "sector":
```

Comment:
Are you sure that the `mode` attribute will be here all the time? If not please add the `hasattr(disk, 'mode')` check.